optimum/intel/openvino/modeling_base.py

```diff
@@ -287,9 +332,21 @@ def _from_pretrained(
             local_files_only=local_files_only,
         )
 
+        compile_only = kwargs.get("compile_only", False)
+
         quantization_config = cls._prepare_weight_quantization_config(quantization_config, load_in_8bit)
 
-        model = cls.load_model(model_cache_path, quantization_config=quantization_config)
+        model = None
+        if not compile_only:
```

Comment:
From my understanding the only thing that `compile_only` does is to skip the `read_model` part is this correct ? To me the argument name `compile_only` is not super clear, but not sure I have a better alternative

Reply:
maybe inference_only? it is the only functional that works in this mode :)

Reply:
Yes I see your point even though might lead users to think the model can be used other than for inference (like training) not sure.  Was thinking about something to highlight that the model cannot be modified, but for now I think we can keep `compile_only` that'll work